piIOComm.h

```diff
@@ -44,6 +44,15 @@
 
 #define REV_PI_TTY_DEVICE	"/dev/ttyAMA0"
 
+typedef enum
+{
+    /* physically not connected */
+    IOSTATE_OFFLINE   = 0x00,
```

Comment:
Here we have again leading spaces, not tabs

Reply:
accept